### PR TITLE
Feature/5 Creating the feedback page

### PR DESCRIPTION
**Pull request by jmezaber**
feedbackPage_demo

related to issue #5

### CODE REVIEW

File: Frontend/moodipy/package.json [hunk 1]
```diff
@@ -10,7 +10,10 @@
   "dependencies": {
     "axios": "^0.21.1",
     "core-js": "^3.6.5",
+    "k-progress": "^1.5.0",
```
Comment (nasaku898) [inline Frontend/moodipy/package.json:13]: unused dependencies

File: Frontend/moodipy/src/components/FeedbackPage/FeedbackPage.vue [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+    <h1 id="title">Your Feedback</h1>
+    <div id="emotions">
+        <p id="emoDetected">Emotions detected</p>
+        <div class="box">
+            <ul>
+                <li v-for="skill in skills" :key="skill.id">
+                    <div class="lang">{{skill.lang}}<span class="percent">({{skill.percent}}%)</span></div>
+                    <div class="bar">
+                        <div class="progress" v-bind:style="{'background':skill.color, 'width':skill.percent+'%'}"></div>
+
```
Comment (nasaku898) [inline Frontend/moodipy/src/components/FeedbackPage/FeedbackPage.vue:12]: white space

File: Frontend/moodipy/package.json (hunk 1 above)
Comment (jmezaber) [reply Frontend/moodipy/package.json]: They have been removed!

File: Frontend/moodipy/src/components/FeedbackPage/FeedbackPage.vue (hunk 2 above)
Comment (jmezaber) [reply Frontend/moodipy/src/components/FeedbackPage/FeedbackPage.vue]: They are gone!

**Review by nasaku898 — APPROVED** [review]

**Pull request merged**